app-backend/datamodel/src/main/scala/Project.scala

```diff
@@ -34,7 +34,8 @@ final case class Project(id: UUID,
                          isSingleBand: Boolean = false,
                          singleBandOptions: Option[SingleBandOptions.Params],
                          defaultAnnotationGroup: Option[UUID],
-                         extras: Option[Json])
+                         extras: Option[Json],
+                         defaultLayer: Option[UUID])
```

Comment:
Given that we're trying to be consistent with projectId and smartLayerId etc, we should probably call this defaultLayerId